soc/arm/st_stm32/stm32g4/soc.c

```diff
@@ -45,6 +49,11 @@ static int stm32g4_init(const struct device *arg)
 	/* allow reflashing board */
 	LL_DBGMCU_EnableDBGSleepMode();
 
+#if defined(PWR_CR3_UCPD_DBDIS)
+	/* Disable USB Type-C dead battery pull-down behavior */
+	__HAL_RCC_PWR_CLK_ENABLE();
```

Comment:
Please use LL API

Reply:
LL api is LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR) from the stm32_ll_bus.h